packages/ckeditor5-dev-tests/bin/save-revision.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const branch = 'master'; // process.env.TRAVIS_BRANCH;
```

Comment:
But we don't want to handle all branches now. Just master. Later on we can add `major` too. The tool should ignore all the other branches. Actually... This is missing now in this PR.